From d8057395f7b68406fac0255c023b68d6a9a1e5f5 Mon Sep 17 00:00:00 2001
From: Chip Hogg <charles.r.hogg@gmail.com>
Date: Sun, 6 Nov 2016 12:55:47 -0500
Subject: [PATCH] Recognize underscores as commit tree characters

Currently, any line in a commit tree representation with an underscore
will not be highlighted correctly.  The solution is to add the
underscore to the list of commit tree characters.

Fixes #766.
---
 syntax/vundlechangelog.vim | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/syntax/vundlechangelog.vim b/syntax/vundlechangelog.vim
index 64e81e32..fc68ea60 100644
--- a/syntax/vundlechangelog.vim
+++ b/syntax/vundlechangelog.vim
@@ -19,7 +19,7 @@ highlight link VundleCompareUrl Underlined
 " The main commit line.
 " Note that this regex is intimately related to the one for VundleCommitTree,
 " and the two should be changed in sync.
-syntax match VundleCommitLine '\v(^  [|*]( *[\\|/\*])* )@<=[^*|].*$'
+syntax match VundleCommitLine '\v(^  [|*]( *[\\|_/\*])* )@<=[^*|].*$'
     \ contains=VundleCommitMerge,VundleCommitUser,VundleCommitTime
 highlight link VundleCommitLine String
 " Sub-regions inside the commit message.
@@ -32,5 +32,5 @@ highlight link VundleCommitTime Comment
 " The git history DAG markers are outside of the main commit line region.
 " Note that this regex is intimately related to the one for VundleCommitLine,
 " and the two should be changed in sync.
-syntax match VundleCommitTree '\v(^  )@<=[|*]( *[\\|/\*])*'
+syntax match VundleCommitTree '\v(^  )@<=[|*]( *[\\|_/\*])*'
 highlight link VundleCommitTree Label
